release/ray_release/scripts/build_pipeline.py

```diff
@@ -242,5 +240,17 @@ def main(
     print(steps_str)
 
 
+def _get_rerun_cmd(
+    test_collection_file: Optional[str] = None,
+    run_jailed_tests: bool = False,
+):
```

Comment:
Do you need to pass `no_clone_repo` as well?

Reply:
not really, that option is not used ;)